templates/etc/init.d/couchdb2.j2

```diff
@@ -0,0 +1,148 @@
+#!/bin/sh
+# Init script for couchdb
+# Maintained by 
+# Generated by pleaserun.
+# Implemented based on LSB Core 3.1:
+#   * Sections: 20.2, 20.3
+#
+### BEGIN INIT INFO
+# Provides:          couchdb
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: 
+# Description:       no description given
+### END INIT INFO
+
+export HOME={{ couchdb_dir }}
+
+PATH=/sbin:/usr/sbin:/bin:/usr/bin
+export PATH
+
+name=couchdb2
+user=couchdb
+program={{ couchdb_dir }}/bin/couchdb
+args=''
+pidfile={{ couchdb_dir }}/var/run/$name.pid
+logdir={{ couchdb_dir }}/var/log
+
+[ -r /etc/default/$name ] && . /etc/default/$name
+[ -r /etc/sysconfig/$name ] && . /etc/sysconfig/$name
+
+trace() {
+    logger -t "/etc/init.d/couchdb" "$@"
+}
+
+emit() {
+    trace "$@"
+    echo "$@"
+}
+
+start() {
+    # Ensure the log directory is setup correctly.
+    [ ! -d "$logdir" ] && mkdir "$logdir"
```

Comment:
please add the creation of the necessary run time directories to the init script somewhere around here!